.github/workflows/ci-dev.yaml

```
@@ -7,7 +7,8 @@ on:
 jobs:
   build-and-push:
     runs-on: ubuntu-latest
-
+    env:
+        SPRING_PROFILES_ACTIVE: test
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 8

(indentation)